integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts

```diff
@@ -1397,34 +1397,46 @@ medusaIntegrationTestRunner({
           })
         })
       })
+
       describe("AddShippingMethodToCartWorkflow", () => {
-        it("should add shipping method to cart", async () => {
-          let cart = await cartModuleService.create({
+        let cart
+        let shippingProfile
+        let fulfillmentSet
+        let priceSet
+
+        beforeEach(async () => {
+          cart = await cartModuleService.create({
             currency_code: "usd",
+            shipping_address: {
+              country_code: "de",
+            },
           })
 
-          const shippingProfile =
-            await fulfillmentModule.createShippingProfiles({
-              name: "Test",
-              type: "default",
-            })
+          shippingProfile = await fulfillmentModule.createShippingProfiles({
+            name: "Test",
+            type: "default",
+          })
 
-          const fulfillmentSet = await fulfillmentModule.create({
+          // TODO: Setting up the geozones here as US, but testing with european countries
+          // in the tests doesn't throw an validations in the module. The context needs to have
+          // some required attributes to test against service zones
+          fulfillmentSet = await fulfillmentModule.create({
```

Comment:
@olivermrbl wdyt?

Reply:
I am not sure I understand. Can I get you to elaborate?

Reply:
nvm, dug a little deeper and its already built in 🎉  - [commit](https://github.com/medusajs/medusa/pull/7150/commits/0f925a76793943fbb8a6cf44d336aeca7ffbd8ed)